include/hax.h

```diff
@@ -232,15 +230,16 @@ static inline unsigned char *hax_page_va(struct hax_page *page)
 #define HAX_MAX_CPUS (sizeof(uint64_t) * 8)
 
 /* Host SMP */
-extern cpumap_t cpu_online_map;
+extern hax_cpumap_t cpu_online_map;
 extern int max_cpus;
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-int smp_call_function(cpumap_t *cpus, void(*scfunc)(void *param), void *param);
-extern int cpu_number(void);
+int hax_smp_call_function(hax_cpumap_t *cpus, void(*scfunc)(void *param),
+                          void *param);
+extern int hax_cpu_number(void);
```

Comment:
We got the following error on Mac when loading the kext:

```
(kernel) kxld[com.intel.kext.intelhaxm]: The following symbols are unresolved for this kext:
(kernel) kxld[com.intel.kext.intelhaxm]:        _hax_cpu_number
(kernel) Can't load kext com.intel.kext.intelhaxm - link failed.
(kernel) Failed to load executable for kext com.intel.kext.intelhaxm.
(kernel) Kext com.intel.kext.intelhaxm failed to load (0xdc008016).
(kernel) Failed to load kext com.intel.kext.intelhaxm (error 0xdc008016).
```

I think `cpu_number()` is probably a function provided by the Darwin kernel. Also, I don't see `hax_cpu_number()` used outside of `platforms/darwin/`, so I'd:

1. Remove this declaration the common header.
2. Keep using `cpu_number()` for Darwin code.

Reply:
Fixed on f153b54.